src/app/bridge/componentWindow.ts

```diff
@@ -0,0 +1,187 @@
+import path from 'path';
+import { fileURLToPath } from 'url';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+interface ComponentWindowOptions {
+  componentName: string;
+  config?: Record<string, unknown>;
+  width?: number;
+  height?: number;
+}
+
+/**
+ * Creates a new Electron window to display a specific component
+ * This allows you to open individual components in separate windows
+ * with full access to telemetry and dashboard bridges
+ * 
+ * Example usage (from main process):
+ *   openComponentWindow({ componentName: 'standings', width: 1024, height: 768 })
+ */
+export function openComponentWindow(options: ComponentWindowOptions): BrowserWindow {
+  const {
+    componentName,
+    config = {},
+    width = 1024,
+    height = 768,
+  } = options;
+
+  const componentWindow = new BrowserWindow({
+    width,
+    height,
+    webPreferences: {
+      preload: path.join(__dirname, '../../preload.ts'),
+      contextIsolation: true,
+      nodeIntegration: false,
+    },
+    show: false,
+  });
+
+  // Pass component info via data URL
+  const componentUrl = `data:text/html;charset=utf-8,${encodeURIComponent(
+    generateComponentWindowHtml(componentName, config)
+  )}`;
+
+  componentWindow.loadURL(componentUrl);
+  componentWindow.show();
+
+  return componentWindow;
+}
+
+/**
+ * Generate HTML for a component window that runs within Electron context
+ */
+function generateComponentWindowHtml(
+  componentName: string,
+  config: Record<string, unknown>
+): string {
+  const configJson = JSON.stringify(config);
+
+  return `<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>${componentName} - iRacing Component</title>
+    <link rel="stylesheet" href="file:///${path.join(__dirname, '../../frontend/index.css')}">
+    <style>
+      * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+      
+      body {
+        font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+          'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+          sans-serif;
+        -webkit-font-smoothing: antialiased;
+        -moz-osx-font-smoothing: grayscale;
+        background: #1a1a1a;
+        color: #fff;
+      }
+      
+      #app {
+        width: 100%;
+        height: 100vh;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+      
+      .loading {
+        font-size: 18px;
+        opacity: 0.7;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="app" class="loading">Loading ${componentName}...</div>
+    <script type="module">
+      import { createRoot } from 'https://esm.sh/react-dom@19';
+      import {
+        TelemetryProvider,
+        DashboardProvider,
+        RunningStateProvider,
+        SessionProvider,
+      } from 'https://esm.sh/@irdashies/context';
```

Comment:
Is there a way to reference the package locally instead? Like the below WidgetIndex? 
I'm not familiar with esm.sh but I suspect this will get you in a world of pain since it would mean the released version would be out of sync with I assume the hosted one would be master.